src/parsers/power_system_table_data.jl

```diff
@@ -819,6 +821,112 @@ function make_thermal_generator(data::PowerSystemTableData, gen, cost_colnames,
     )
 end
 
+function make_thermal_generator_multistart(
+    data::PowerSystemTableData,
+    gen,
+    cost_colnames,
+    bus,
+)
+    fuel_cost = gen.fuel_price / 1000
+
+    var_cost = [(getfield(gen, hr), getfield(gen, mw)) for (hr, mw) in cost_colnames]
```

Comment:
Is the calculation of var_cost a direct copy/paste of the similar code in make_thermal_generator?  If so, could you make it a common function?

Reply:
Sure